feature-group-definitions/webhid.yml

```diff
@@ -0,0 +1,36 @@
+spec: https://wicg.github.io/webhid/
```

Comment:
It might be nice to have a TODO comment or an issue noting that this is a… conflicted specification. That is, Apple and Mozilla are explicitly _not_ prototyping/implementing the spec. That's an interesting bucket that I imagine we'll get to eventually and it'd be nice to have a way to come back to it when we're ready to add it to the schema.

Reply:
I think the best way to capture that would be a `standards_positions` field that can link to issues in https://github.com/mozilla/standards-positions or https://github.com/WebKit/standards-positions.

Reply:
Tracked in https://github.com/web-platform-dx/feature-set/issues/186